### DISCUSSION TITLE
How do I create a file for blocklist when starting self control from the command line (v4)

### DISCUSSION THREAD

**Discussion by lizard-heart** [SelfControlApp/selfcontrol#744] [Q&A] [answered]
I want to be able to start self control from the terminal, but I keep getting this error: `ERROR: Block could not be read from file my_blocklist.selfcontrol`.

What is that blocklist file supposed to be? Does a blocklist file already exist if I've been using Self Control in the GUI or do I have to create a new one?

**Comment by cstigler** [comment, marked as answer]
Hi @lizard-heart , you can save your blocklist from the SelfControl app with the File --> Save Blocklist option, or Cmd-S

**Reply by lizard-heart** [reply]
Thank you! I probably should have figured this out on my own.
